client/src/components/v2/CameraRecordingStatus.module.css

```diff
@@ -0,0 +1,11 @@
+.bottomMargin {
```

Comment:
Class names that relate to the component, rather than the effect applied, are preferred. It's a bit more descriptive and obvious when looking at the CSS and figuring out what-does-what

Reply:
Done
869d24f